config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-presubmits.yaml

```diff
@@ -1,61 +1,28 @@
 presubmits:
   kubernetes-sigs/cluster-api-provider-gcp:
-  - name: pull-cluster-api-provider-gcp-build
-    always_run: true
-    labels:
-      preset-service-account: "true"
-    spec:
-      containers:
-      - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
-        args:
-        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
-        - "--root=/go/src"
-        - "--upload=gs://kubernetes-jenkins/pr-logs"
-        - "--scenario=execute"
-        - "--"
-        - "./scripts/ci-build.sh"
-    annotations:
-      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
-      testgrid-tab-name: pr-build
-  - name: pull-cluster-api-provider-gcp-make
```

Comment:
The make target isn't actually doing anything differently than build, so we remove it

Reply:
It looks like scripts/ci-make.sh is calling `make docker-build` and scripts/ci-build.sh is calling `make manager clusterctl`, so I'm not sure these should be de-duped currently.

Reply:
Not sure what I was looking at, fixed!